pkg/errdefs/wrap.go

```diff
@@ -0,0 +1,37 @@
+// A wrapper around pkg/errors.
+// Prints the error call stack when the environment variable DEBUG is on, otherwise it does not print
+package errdefs
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+)
+
+var debug bool
+
+func init() {
+	if os.Getenv("DEBUG") == "enable" {
```

Comment:
1/0 should be enough.

`DEBUG=1 ...`

Reply:
updated. `DEBUG` equal `true`, `1`, `enable` will print err stack.